### PR TITLE
fix(cli): ensure empty lines don't count towards coverage

### PR DESCRIPTION
**Pull request by caspervonb**
This cleans up coverage by computing the result ahead of time and passing that as-is into the reporter(s).
Doing so, I made sure to throw out empty lines.

Fixes https://github.com/denoland/deno/issues/11327

### CODE REVIEW

**Comment by caspervonb** [comment]
All of the mapping and lookups in this is a prime candidate to make use of the denoland/deno_ast utilities.

File: cli/tools/coverage.rs [hunk 1]
```diff
@@ -697,24 +430,259 @@ pub async fn cover_files(
...
+          offset_to_line_col(&compiled_source, line_range.start_offset)
+            .unwrap();
+
+        let start_token =
+          source_map.lookup_token(start_line, start_col).unwrap();
+
+        let (end_line, end_col) =
+          offset_to_line_col(&compiled_source, line_range.end_offset).unwrap();
+
+        let end_token = source_map.lookup_token(end_line, end_col).unwrap();
```
Comment (caspervonb) [inline cli/tools/coverage.rs:505]: These end up being too short.

Comment (bartlomieju) [reply cli/tools/coverage.rs:505]: @dsherret can you advise?

**Review by bartlomieju — APPROVED** [review]
LGTM, thank you and sorry it took so long to land.

**Pull request merged**